### PR TITLE
Adds a mentor ban command

### PR DESCRIPTION
**Pull request by LoliconSlayer**
Ready to go, only people with the Mentor 'Council' role can use the command as per Mqiib's request, if you have issue with that go talk to him please do not speak a word of it to me

### CODE REVIEW

**Comment by N3D6** [comment]
Thank you. So much. Speedmerge!!!

**Comment by Cartlord** [comment]
holy shit it's actually happening

**Review by adamsong — COMMENTED** [review]
Code looks good, just needs the role id

**Comment by adamsong** [comment]
Is a role going to be added?

File: data/discord.json [hunk 1]
```diff
@@ -32,6 +32,10 @@
       "inherits": false,
       "permissions": ["mhelp"]
     },
+    "Mentor 'Council'": {
+      "inherits": false,
```
Comment (TheGamerdk) [inline data/discord.json:36]: Make it inherit from mentor

**Review by TheGamerdk — CHANGES_REQUESTED** [review]

File: data/discord.json (hunk 1 above)
Comment (LoliconSlayer) [reply data/discord.json:36]: No.

Comment (TheGamerdk) [reply data/discord.json:36]: Yes.

Comment (TheGamerdk) [inline data/discord.json:36]:
```suggestion
      "inherits": “mentor”,
```

**Review by TheGamerdk — CHANGES_REQUESTED** [review]

File: data/discord.json (hunk 1 above)
Comment (LoliconSlayer) [reply data/discord.json:36]: No.

Comment (TheGamerdk) [inline data/discord.json:36]:
```suggestion
      "inherits": “mentor”,
```

Comment (LoliconSlayer) [reply data/discord.json:36]: No.

Comment (TheGamerdk) [inline data/discord.json:36]:
```suggestion
      "inherits": ”mentor”,
```

**Review by adamsong — APPROVED** [review]
Does the needful

**Pull request merged**